### DISCUSSION TITLE
Help needed on how to install Typst and its packages (like cetz) on ubuntu

### DISCUSSION THREAD

**Discussion by malnaanah** [typst/typst#3923] [Q&A]
I started to use Typst in Xournal++ and it is really good. I found out that it has a plugin for drawing called Cetz, and I can't figure out how to install it.
So far, I downloaded the binary file and extracted it, and put its path in Xournal++.

I would appreciate a short tutorial on how to install Typst in Ubuntu, and how to install its plug. Also, how to instal the plugins if I just have the binary file copied at some directory, like /opt/typst.

Best ragrads,

**Comment by laurmaedje** [comment]
The packages are automatically downloaded on demand. If you copy an example from https://typst.app/universe/package/cetz/, it should just work.

**Comment by malnaanah** [comment]
Thank you, it works.
